### PR TITLE
Le potato patch

### PR DESCRIPTION
**Pull request by Iainrach**
I'm really not sure if I've done this correctly but this is related to issue 654.

### CODE REVIEW

File: src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py [hunk 1]
```diff
@@ -53,17 +53,17 @@
...
 UART2_TX = GPIO291
 
-SPI1_SCLK = GPIO223
-SPI1_MISO = GPIO288
-SPI1_MOSI = GPIO287
-SPI1_CS = GPIO289
+SPI0_SCLK = GPIO290
+SPI0_MISO = GPIO288
+SPI0_MOSI = GPIO287
+SPI0_CS = GPIO289
```
Comment (makermelissa) [inline src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py:59]:
```suggestion
SPI0_SCLK = GPIO290
SPI0_MISO = GPIO288
SPI0_MOSI = GPIO287
SPI0_CS = GPIO289
SPI1_SCLK = GPIO223
SPI1_MISO = GPIO288
SPI1_MOSI = GPIO287
SPI1_CS = GPIO289
```

File: src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py [hunk 2]
```diff
@@ -53,17 +53,17 @@
...
 
 i2cPorts = (
     (0, I2C0_SCK, I2C0_SDA),
     (1, I2C1_SCK, I2C1_SDA),
 )
 
-spiPorts = (0, SPI1_SCLK, SPI1_MOSI, SPI1_MISO)
+spiPorts = [(0, SPI0_SCLK, SPI0_MOSI, SPI0_MISO)]
```
Comment (makermelissa) [inline src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py:66]:
```suggestion
spiPorts = (
    (0, SPI0_SCLK, SPI0_MOSI, SPI0_MISO),
    (1, SPI1_SCLK, SPI1_MOSI, SPI1_MISO)
)
```

**Review by makermelissa — CHANGES_REQUESTED** [review]
Hi, thanks for adding. You're pretty close. You should be able to keep the SPI1 pins. I added some suggested code changes to achieve this so that it should still work with your board.

File: src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py (hunk 2 above)
Comment (Iainrach) [reply src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py:70]: https://learn.adafruit.com/adding-a-single-board-computer-to-blinka/getting-spi-working

From the link above 

At this time, only Spidev 0 is supported. ??

Comment (makermelissa) [reply src/adafruit_blinka/microcontroller/amlogic/s905x/pin.py]: Yeah, meaning spidev0.0 and spidev0.1, but not spidev1.0 or spidev1.1. I'll try and make that a bit clearer. :)

**Review by makermelissa — APPROVED** [review]
Thanks

**Pull request merged**